app/src/main/java/android/coding/interview/makeitawesome/fragment/PicturesFragment.java

```diff
@@ -19,19 +30,78 @@
 public class PicturesFragment extends Fragment {
 
     public static Fragment newInstance() {
-        return new PicturesFragment();
+        PicturesFragment pictures = new PicturesFragment();
+        pictures.setRetainInstance(true);
```

Comment:
This is not a good practice, but you can't completely blame someone for using it. At least, in this version he worries about the lifecycle.